plugin/src/test/java/org/opensearch/ml/rest/RestMLFlowAgentIT.java

```diff
@@ -9,11 +9,26 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.hc.core5.http.ParseException;
+import org.junit.After;
+import org.junit.Before;
 import org.opensearch.client.Response;
 import org.opensearch.ml.utils.TestHelper;
 
 public class RestMLFlowAgentIT extends MLCommonsRestTestCase {
 
+    private String irisIndex = "iris_data";
+
+    @Before
+    public void setup() throws IOException, ParseException {
+        ingestIrisData(irisIndex);
```

Comment:
Is it possible to merge this IT with RestMLTrainAndPredictIT, to save time in the ingesting irisIndex before and delete it after?

Reply:
I prefer to have it in agent IT, as it is for agent framework testing. And it can be reused for other agent test cases.

Reply:
LGTM for this test. But I think we can delete RestMLTrainAndPredictIT, and move the tests with new names inside this agentIT to save IT time.

Reply:
In my opinion we still need to separate the ITs by its category unless there is obvious concern due to maintainability. Consuming same index doesn't necessarily mean they need to be merged together.